### DISCUSSION TITLE
I got a lhv whoes  frontend mode is blockdev ,but has a iscsi session,why this happen?

### DISCUSSION THREAD

**Discussion by adababys** [longhorn/longhorn#6054] [General]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.

## Environment

 - Longhorn version:1.4.1
 - Kubernetes version:1.23.9
 - Node config
   - OS type and version openeuler 4.18.0-372.9.1.15.pol.x86_64
   - CPU per node:
   - Memory per node:
   - Disk type:
   - Network bandwidth and latency between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):

## Additional context

# iscsiadm -m session
tcp: [30] 184.10.12.158:3260,1 [iqn.2019-10.io](http://iqn.2019-10.io/).longhorn:pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e (non-flash)

 # kubectl describe lhv -n longhorn-system pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e
Name: pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e
Namespace: longhorn-system
Labels: longhornvolume=pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e
[recurring-job-group.longhorn.io/default=enabled](http://recurring-job-group.longhorn.io/default=enabled)
[setting.longhorn.io/remove-snapshots-during-filesystem-trim=ignored](http://setting.longhorn.io/remove-snapshots-during-filesystem-trim=ignored)
[setting.longhorn.io/replica-auto-balance=ignored](http://setting.longhorn.io/replica-auto-balance=ignored)
[setting.longhorn.io/snapshot-data-integrity=ignored](http://setting.longhorn.io/snapshot-data-integrity=ignored)
Annotations: <none>
API Version: [longhorn.io/v1beta2](http://longhorn.io/v1beta2)
Kind: Volume
Metadata:
Creation Timestamp: 2023-05-08T08:56:17Z
Finalizers:
[longhorn.io](http://longhorn.io/)
Generation: 35
Managed Fields:
API Version: [longhorn.io/v1beta2](http://longhorn.io/v1beta2)
Fields Type: FieldsV1
fieldsV1:
f:status:
.:
f:actualSize:
f:cloneStatus:
.:
f:snapshot:
f:sourceVolume:
f:state:
f:conditions:
f:currentImage:
f:currentNodeID:
f:expansionRequired:
f:frontendDisabled:
f:isStandby:
f:kubernetesStatus:
.:
f:lastPVCRefAt:
f:lastPodRefAt:
f:namespace:
f:pvName:
f:pvStatus:
f:pvcName:
f:workloadsStatus:
f:lastBackup:
f:lastBackupAt:
f:lastDegradedAt:
f:ownerID:
f:pendingNodeID:
f:remountRequestedAt:
f:restoreInitiated:
f:restoreRequired:
f:robustness:
f:shareEndpoint:
f:shareState:
f:state:
Manager: longhorn-manager
Operation: Update
Subresource: status
Time: 2023-05-08T08:56:17Z
API Version: [longhorn.io/v1beta2](http://longhorn.io/v1beta2)
Fields Type: FieldsV1
fieldsV1:
f:metadata:
f:finalizers:
.:
v:"[longhorn.io](http://longhorn.io/)":
f:labels:
.:
f:longhornvolume:
f:[recurring-job-group.longhorn.io/default](http://recurring-job-group.longhorn.io/default):
f:spec:
.:
f:Standby:
f:accessMode:
f:backingImage:
f:baseImage:
f:dataLocality:
f:dataSource:
f:disableFrontend:
f:encrypted:
f:engineImage:
f:fromBackup:
f:frontend:
f:lastAttachedBy:
f:migratable:
f:migrationNodeID:
f:nodeID:
f:numberOfReplicas:
f:replicaAutoBalance:
f:restoreVolumeRecurringJob:
f:revisionCounterDisabled:
f:size:
f:snapshotDataIntegrity:
f:staleReplicaTimeout:
f:unmapMarkSnapChainRemoved:
Manager: longhorn-manager
Operation: Update
Time: 2023-05-10T06:43:43Z
Resource Version: 16705555
UID: bfe2521d-0808-4e8f-9de6-6d8afbd02296
Spec:
Standby: false
Access Mode: rwx
Backing Image:
Base Image:
Data Locality: disabled
Data Source:
Disable Frontend: false
Disk Selector:
Encrypted: false
Engine Image: proxy.harbor/local-xdr/longhorn-engine:v1.4.1
From Backup:
Frontend: blockdev
Last Attached By:
Migratable: false
Migration Node ID:
Node ID: dmoc-fc73fb8e9abd
Node Selector:
Number Of Replicas: 3
Recurring Jobs:
Replica Auto Balance: ignored
Restore Volume Recurring Job: ignored
Revision Counter Disabled: false
Size: 26843545600
Snapshot Data Integrity: ignored
Stale Replica Timeout: 30
Unmap Mark Snap Chain Removed: ignored
Status:
Actual Size: 25859387392
Clone Status:
Snapshot:
Source Volume:
State:
Conditions:
Last Probe Time:
Last Transition Time: 2023-05-08T08:56:17Z
Message:
Reason:
Status: False
Type: toomanysnapshots
Last Probe Time:
Last Transition Time: 2023-05-08T08:56:17Z
Message:
Reason:
Status: True
Type: scheduled
Last Probe Time:
Last Transition Time: 2023-05-08T08:56:17Z
Message:
Reason:
Status: False
Type: restore
Current Image: proxy.harbor/local-xdr/longhorn-engine:v1.4.1
Current Node ID: dmoc-fc73fb8e9abd
Expansion Required: false
Frontend Disabled: false
Is Standby: false
Kubernetes Status:
Last PVC Ref At:
Last Pod Ref At:
Namespace: xdrsec
Pv Name: pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e
Pv Status: Bound
Pvc Name: alpha-app-data-10001001
Workloads Status:
Pod Name: alpha-app-10001001-dbddd86d-7j6hv
Pod Status: Running
Workload Name: alpha-app-10001001-dbddd86d
Workload Type: ReplicaSet
Pod Name: alpha-app-10001001-taskmanager-2-3
Pod Status: Running
Workload Name: alpha-app-10001001
Workload Type: Deployment
Last Backup:
Last Backup At:
Last Degraded At:
Owner ID: dmoc-fc73fb8e9abd
Pending Node ID:
Remount Requested At: 2023-05-31T08:30:02Z
Restore Initiated: false
Restore Required: false
Robustness: healthy
Share Endpoint: nfs://186.30.197.56/pvc-340feaeb-6d23-40a7-b9af-41fdbf571e6e
Share State: running
State: attached
Events:
Type Reason Age From Message

**Comment by ejweber** [comment]
Hello @shaolila, even when using the block device frontend, the Longhorn engine exposes itself as an iSCSI target and workloads communicate with it via an iSCSI initiator. iSCSI is always involved when using Longhorn.
